### PR TITLE
Added the functionality to the insert column dialog. Works, but still needs improvement. Working with Steve on that.

### PR DESCRIPTION
**Pull request by deaspo**
Added the functionality to the insert column dialog. Works, but still needs improvement. Working with Steve on that.

### CODE REVIEW

File: instat/dlgInsertColumn.vb [hunk 1]
```diff
@@ -1,3 +1,69 @@
...
+            setDefaultValues()
+            bFirstLoad = False
+        End If
+        ucrBase.clsRsyntax.SetFunction(frmMain.clsRLink.strInstatDataObject & "$insert_column_in_data")
+    End Sub
+
+    Private Sub setDefaultValues()
+        txtDefaultData.Text = "NA"
+        txtStartPos.Text = "At the end."
+        ucrBase.clsRsyntax.AddParameter("start_pos ", "length(" & ucrDataFramesList.cboAvailableDataFrames.SelectedItem & ")")
```
Comment (dannyparsons) [inline instat/dlgInsertColumn.vb]: You can use iDataFrameLength in ucrDataFramesList instead of using length.

Comment (dannyparsons) [inline instat/dlgInsertColumn.vb]: Remove the space after start_pos

File: instat/dlgInsertColumn.vb [hunk 2]
```diff
@@ -1,3 +1,69 @@
...
+        nudPos.Enabled = False
+    End Sub
+
+    Private Sub ucrDataFramesList_Leave(sender As Object, e As EventArgs) Handles ucrDataFramesList.Leave
+        ucrBase.clsRsyntax.AddParameter("data_name", Chr(34) & ucrDataFramesList.cboAvailableDataFrames.SelectedItem & Chr(34))
+    End Sub
+
+    Private Sub txtDefaultData_Leave(sender As Object, e As EventArgs) Handles txtDefaultData.Leave
+        ucrBase.clsRsyntax.AddParameter("col_data ", "c(" & txtDefaultData.Text & ")")
```
Comment (dannyparsons) [inline instat/dlgInsertColumn.vb]: I think get rid of txtDefaultData for now to keep it simple and always pass in col_data = c() for now.
But it's a good point to discuss whether we should be able to insert a column with data in it and not just a blank one.

File: instat/dlgInsertColumn.vb (hunk 1 above)
Comment (deaspo) [inline instat/dlgInsertColumn.vb]: iDataFrameLength gives me the number of rows and not columns, which in this case is what I am after, inserting at the last column.

**Comment by deaspo** [comment]
I have added those changes.

**Pull request merged**